src/editor/EditorCommandHandlers.js

```diff
@@ -207,7 +209,9 @@ define(function (require, exports, module) {
         if (containsNotLineComment) {
             // Comment out - prepend the first prefix to each line
             for (i = startLine; i <= endLine; i++) {
-                editGroup.push({text: prefixes[0], start: {line: i, ch: 0}});
+                // For each line we need to determine the position where de comment prefix should be inserted
+                prefixPos = _getPrefixPos(language, doc.getLine(i));
```

Comment:
The `language` var was removed from this function more recently, so you'll need to re-add it